scylla/src/session/mod.rs

```diff
@@ -1,3 +1,14 @@
+//! This module holds entities that represent the whole configurable
+//! driver session with the cluster.
+//! The following abstractions are involved:
+//! - Session - the main entity of the driver. It:
+//!     - contains and manages all driver configuration,
+//!     - launches and communicates with ClusterWorker (see cluster module for more info),
+//!     - enables executing CQL requests, taking all configuration into consideration.
+//! - SessionBuilder - just a convenient builder for a Session.
+//! - CachingSession - a wrapper over a Session that keeps and manages a cache
+//!   of prepared statements, so that a user can be free of such considerations.
+
```

Comment:
Commit: "execution: add module docstring"

🔧 Commit message seems to be wrong - commit adds doc comment for session, not for execution.